### PR TITLE
Feat_server_health_info

### PR DESCRIPTION
**Pull request by Funmi4194**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,14 +0,0 @@
...
-
-build:
-	go build -o bin/$(NAME) -ldflags "-X main.Version=$(VERSION)" $(NAME).go
-
-run:
-	make build
-	./bin/$(NAME)
-
-start:
-	./bin/$(NAME)
```
Comment (opensaucerer) [inline Makefile:14]: As in the previous PR comment. Please let's restore the `Makefile`

File: version/v1.go [hunk 2]
```diff
@@ -8,17 +8,11 @@ import (
 // Version1Routes registers all routes for the v1 version
 func Version1Routes(r *mux.Router) {
 
-	// this doesn't need versioning, yet
+	//server info in json format
 	base.RegisterHealthRoutes(r)
```
Comment (opensaucerer) [inline version/v1.go:12]: Why do we have two `base.RegisterHealthRoutes(r)` one on `line 12` and another on `line 17`?

**Review by opensaucerer — COMMENTED** [review]
Left a few comments

File: version/v1.go (hunk 2 above)
Comment (Funmi4194) [reply version/v1.go:12]: You said the first endpoint should be "/" while the second should be "v1/health".  If I use just base.RegisterHealthRoutes(route) in line 17, the first enpoint would be now be "/v1". Should it be done that way?

File: Makefile (hunk 1 above)
Comment (Funmi4194) [reply Makefile:14]: Okay. Can I replace "raidtoearn" in Makefile to "Golang server template"?

Comment (opensaucerer) [reply Makefile:14]: replace it with `app`

Comment (Funmi4194) [reply Makefile:14]: okay

File: version/v1.go (hunk 2 above)
Comment (opensaucerer) [reply version/v1.go:12]: I'm wondering why they are both health routes. One is health and the other is home. So, why group them the same way?

Comment (Funmi4194) [reply version/v1.go:12]: I had no idea I was supposed to group them separately, I will correct it. 
Thank you.

Comment (Funmi4194) [reply version/v1.go:12]: In controller folder, what can I call the name of the handler for the first endpoint? I named the second handler as `HealthInfo`

Comment (Funmi4194) [reply version/v1.go:12]: For now, I left it as `HealthCheck`

Comment (opensaucerer) [reply version/v1.go:12]: > I'm wondering why they are both health routes. One is health and the other is home. So, why group them the same way?

here's my message above...I think you find a name from it.

Comment (Funmi4194) [reply version/v1.go:12]: okay

Comment (opensaucerer) [reply version/v1.go:12]: one is Health and one is Home

```go
func HealthCheck() {}

func Home() {}
```

Comment (Funmi4194) [reply version/v1.go:12]: Done that, Thank youu.

**Pull request closed without merging**